modules/network-observability-viewing-network-events.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+//
+// * network_observability/observing-network-traffic.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="network-observability-viewing-network-events_{context}"]
+= Viewing network events
+You can view network events, such as network flows that are dropped or allowed by a network policy, by editing the `FlowCollector` to the specifications in the following YAML example.
+
+.Prerequisites
+* Must have the `OVNObservability` specified in the `FeatureGate` custom resource. For more information, see "Enabling features using feature gates" in the "Additional resources" of this section.
+
+.Procedure
+. In the web console, navigate to *Operators* -> *Installed Operators*.
+. In the *Provided APIs* heading for the *NetObserv Operator*, select *Flow Collector*.
+. Select *cluster*, and then select the *YAML* tab.
+. Configure the `FlowCollector` custom resource to enable viewing `NetworkEvents`, for example:
++
+[id="network-observability-flowcollector-configuring-networkevents{context}"]
+.Example `FlowCollector` configuration
+[source, yaml]
+----
+apiVersion: flows.netobserv.io/v1beta2
+kind: FlowCollector
+metadata:
+  name: cluster
+spec:
+   sampling: 1
+      privileged: true
+      features:
+       - "NetworkEvents"
+----
+
+.Verification
+. Navigate to the *Network Traffic* view and select the *Traffic flows* table. 
+. You should see the new column, *Network Events*, where you can view information about impacts that network policies, admin policies, and egress firewalls have on network flows.
```

Comment:
It would be great to add example of output messages here
cc @msherif1234

Reply:
Thanks for the suggestion, @jpinsonneau  Do you mean an example of what's printed in the cell of the Network Events column? Or another kind of output message?

Reply:
yes example of informations this column can provide since it cover tons of different cases

Reply:
@jpinsonneau awesome, thanks for the idea. I added these based on a screencapture that @msherif1234 provided of the column in the console. https://github.com/openshift/openshift-docs/pull/80846/files#diff-2c58e02f3f3d88ed7f2bea54b85dd8efb8d183028d9c8ba19f0721704f933109R39-R54

Reply:
@msherif1234 do we have a full list of these ?

I only found https://github.com/ovn-org/ovn-kubernetes/pull/4596/files#diff-f3bfaffd193f40c714d0dd4f96636c614bd73cd4d1c12cbcd1ebdc31e4041e56R161 and the related tech doc https://github.com/ovn-org/ovn-kubernetes/pull/4596/files#diff-2ec9b512514bf19c87313b7da12c9436f73d963ece18048a4372b6af918124ca mentionning:
```
Currently, supports observability for:
- Network Policy
- (Baseline) Admin Network Policy
- Egress firewall
- UDN isolation
- Multicast ACLs
```

That could be added too ?